cocos/audio/android/IAudioPlayer.h

```diff
@@ -0,0 +1,89 @@
+/****************************************************************************
+Copyright (c) 2016 Chukong Technologies Inc.
+
+http://www.cocos2d-x.org
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+****************************************************************************/
+
+#ifndef COCOS_IAUDIOPLAYER_H
+#define COCOS_IAUDIOPLAYER_H
+
+#include <functional>
+
+namespace cocos2d { namespace experimental {
+
+class IAudioPlayer
+{
+public:
+    virtual ~IAudioPlayer()
+    { };
+
+    virtual int getId() const = 0;
+
+    virtual void setId(int id) = 0;
+
+    virtual std::string getUrl() const = 0;
+
+    enum class State
+    {
+        INVALID = 0,
+        INITIALIZED,
+        PLAYING,
+        PAUSED,
+        STOPPED,
+        OVER
+    };
```

Comment:
DONE at https://github.com/dumganhar/cocos2d-x/blob/audio-latency-fix/cocos/audio/android/IAudioPlayer.h#L42

I don't know why `github` still show this comment. :(